src/service/service.ts

```diff
@@ -83,18 +79,14 @@ class Service {
 }
 
 interface Tasks {
-  [task: string]: (inputs: TaskInputs, outputs: TaskOutputs) => void
+  [task: string]: (inputs: TaskInputs) => TaskOutputs
 }
 
 interface TaskInputs {
   [key: string]: any
 }
 
 interface TaskOutputs  {
-  [key: string]: (input: TaskOutputCallbackInput) => Promise<void>
```

Comment:
how about having `Promise<object>` as the return type. because with the updated syntax it's unkown for devs if output type can be a promise.

Reply:
agree but see comment https://github.com/mesg-foundation/mesg-js/pull/90#discussion_r287338803